mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -58,8 +58,11 @@ private InputFormatConfig() {
   public static final String LOCATION_PROVIDER = "iceberg.mr.location.provider";
   public static final String ENCRYPTION_MANAGER = "iceberg.mr.encription.manager";
 
-  public static final String COMMIT_THREAD_POOL_SIZE = "iceberg.mr.commit.thread.pool.size";
-  public static final int COMMIT_THREAD_POOL_SIZE_DEFAULT = 10;
+  public static final String OUTPUT_TABLES = "iceberg.mr.output.tables";
+  public static final String COMMIT_TABLE_THREAD_POOL_SIZE = "iceberg.mr.commit.table.thread.pool.size";
+  public static final int COMMIT_TABLE_THREAD_POOL_SIZE_DEFAULT = 10;
```

Comment:
What's the average number of tables we'll write to in a typical insert/multitable-insert? Isn't it typically a lot lower than 10? I'm worried we'll pay a performance penalty for starting a threadpool with at least 10 threads every time we perform a write, even when we write to a single table only (which I'd think is the most common scenario).

Reply:
Yeah... wanted to use `size = Math.min(outputSize, size);` instead of `size = Math.max(outputSize, size);` in `tableExecutor` 😄

Reply:
Would you still change this if we use min?

Reply:
No, with the min it makes sense :)